_docs/applications/postgres.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: guide
-title: StorageOS Docs - PostgreSQL
+title: PostgreSQL with StorageOS
```

Comment:
This is not consistent - I'll change the other pages under applications to reflect this as its just MS SQL and Postgres currently.